includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -405,11 +407,12 @@ protected function get_row_data( $item ) {
 				$column_data = apply_filters(
 					'sensei_analysis_overview_column_data',
 					array(
-						'title'         => $lesson_title,
-						'course'        => $course_title,
-						'students'      => $lesson_students,
-						'completions'   => $lesson_completions,
-						'average_grade' => $lesson_average_grade,
+						'title'           => $lesson_title,
+						'course'          => $course_title,
+						'students'        => $lesson_students,
+						'completions'     => $lesson_completions,
+						'completion_rate' => $lesson_students > 0 ? round( ( $lesson_completions / $lesson_students ) * 100 ) . '%' : __( 'n/a', 'sensei-lms' ),
```

Comment:
What do you think about making this logic a bit easier to read? For example splitting it on multiple lines:

```php
$completion_rate = __( 'n/a', 'sensei-lms' );
if ( $lesson_students ) {
	$completion_rate = round( ( $lesson_completions / $lesson_students ) * 100 ) . '%';
}
```

This is just a preference thing, so feel free to leave it as-is if you like. 🙂

Reply:
On a side note, we decided to use capitalized "N/A".

Reply:
[As for testing](https://github.com/Automattic/sensei/pull/4897#pullrequestreview-899479122), it might be helpful to extract this expression in its own method. Otherwise, testing looks challenging :)